lib/http.c

```diff
@@ -650,7 +650,7 @@ CURLcode Curl_http_auth_act(struct Curl_easy *data)
   if(!data->set.str[STRING_BEARER])
     authmask &= (unsigned long)~CURLAUTH_BEARER;
 
-  if(100 <= data->req.httpcode && 199 >= data->req.httpcode)
+  if(100 <= data->req.httpcode && data->req.httpcode <= 199)
```

Comment:
Not sure what this is supposed to fix. The rest looks good to me!

Reply:
This one isn't supposed to fix anything it's just a style change.